client/scripts/controllers/server/user/base.ts

```diff
@@ -18,6 +18,10 @@ export default class {
   public get activeAccount(): Account<any> { return this._activeAccount; }
   private _setActiveAccount(account: Account<any>): void { this._activeAccount = account; }
 
+  private _user_id: number;
```

Comment:
What does userId do?

Reply:
I need the user ID to fetch the ghost address
```
const { id: ghostAddressId } = await models.Address.scope('withPrivateData').findOne({
        where: { chain: 'injective', ghost_address: true, user_id: userId }
```